src/control/security/pem.go

```diff
@@ -21,9 +21,10 @@ import (
 )
 
 const (
-	MaxKeyPerm  os.FileMode = 0700
-	MaxCertPerm os.FileMode = 0664
-	MaxDirPerm  os.FileMode = 0700
+	MaxUserOnlyKeyPerm os.FileMode = 0400
+	MaxGroupKeyPerm    os.FileMode = 0440
+	MaxCertPerm        os.FileMode = 0664
+	MaxDirPerm         os.FileMode = 0700
```

Comment:
Probably a stupid question (not really familiar with this portion of the code).
Is the key file stored in the directory with MaxDirPerm ?
If yes, even with MaxGroupKeyPerm set, the user from the authorized group will not be able to access the key cert ?

Reply:
I agree, I think MaxDirPerm should be 0750 or "group" cannot get to the admin.key

Reply:
Good catch. Fixing